tests/playwright/accessibility-tests/search-page.spec.ts

```diff
@@ -32,6 +32,32 @@ test.describe('search page should not have any automatically detectable accessib
     await expectNoAccessibilityViolations()
   })
 
+  test('when there are multiple pages results you are able to navigate the results', async ({ expectNoAccessibilityViolations }) => {
+    await searchPage.goToPageWithManyResults()
+    await searchPage.pagination.expect.toBeOnSpecificPage(1)
+    await searchPage.pagination.expect.toNotShowPreviousPageLink()
+    await searchPage.pagination.expect.toShowNextPageLink()
+    await searchPage.pagination.selectNextPage()
+    await searchPage.pagination.expect.toBeOnSpecificPage(2)
```

Comment:
Just wondering why we are testing the specific pages here? Are you hoping to check for no accessibility violations in each scenario? If so as far as I am aware we need to add `await expectNoAccessibilityViolations()` at each point

Reply:
I think this might be me not understanding how that function works. I wanted to do it on each page so it checked all the new pagination buttons, since some are hidden when there is not a previous or next page. I'll change it to run the accessibility check after each page change.